### DISCUSSION TITLE
the http-request node lose  json requset data after i saved it

### DISCUSSION THREAD

**Discussion by AngleMAXIN** [langgenius/dify#4123] [Help]
url: https://cloud.dify.ai/app/f79e94f0-1ad9-47eb-a224-a03297f4e0c2/workflow
first, all http request is ok, request body include my json data, but lose it after refresh it , even i click publish button

![image](https://github.com/langgenius/dify/assets/30991830/b689c7f2-3d03-451e-a229-91d350bd6825)
click again ...
![image](https://github.com/langgenius/dify/assets/30991830/9ba575bc-8883-4f16-8b13-92d1075016e1)

**Comment by patryk20120** [comment]
Probably will be fixed with https://github.com/langgenius/dify/pull/4127

**Reply by AngleMAXIN** [reply]
When will it be published to cloud.dify.ai? it's an extremely problem i think, please resolve it

**Comment by AngleMAXIN** [comment]
the image version langgenius/dify-api:0.6.5 is ok, langgenius/dify-api:0.6.6 is not ok
